modules/test/playwright/tests/object-web/objectFields.spec.ts

```diff
@@ -43,6 +48,18 @@ test.afterEach(async ({apiHelpers}) => {
 		createdEntities.objectDefinitions = [];
 	}
 
+	if (createdEntities.objectFolders.length) {
+		await Promise.all(
+			createdEntities.objectFolders.map(async (objectFolder) => {
+				await apiHelpers.objectAdmin.deleteObjectFolder(
+					objectFolder.id
+				);
+			})
+		);
```

Comment:
Hey @igor-franca, I think we can use the `AsyncArray` class here too, could you try?